### PR TITLE
Give @oleorhagen even more privilege

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @vpodzime can review this?

**Comment by oleorhagen** [comment]
Feed me a task @vpodzime ;)

File: config.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     {
       "username": "cf-bottom",
       "secrets": "cfengine-secrets.json",
-      "trusted": ["nickanderson", "vpodzime", "craigcomstock", "olehermanse", "Lex-2008", "karlhto"],
+      "trusted": ["nickanderson", "vpodzime", "craigcomstock", "olehermanse", "Lex-2008", "karlhto", "oleorhagen"],
```
Comment (karlhto) [inline config.json]: Unnecessary change :^)

File: config.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
         "cfengine/buildscripts": ["craigcomstock", "Lex-2008"],
         "cfengine/documentation": ["nickanderson", "craigcomstock"],
         "cfengine/contrib": ["nickanderson"],
-        "cf-bottom/self": ["olehermanse", "vpodzime", "cf-bottom", "karlhto"]
+        "cf-bottom/self": ["olehermanse", "vpodzime", "cf-bottom", "karlhto", "oleorhagen"]
```
Comment (karlhto) [inline config.json]: ^

File: config.json (hunk 1 above)
Comment (karlhto) [reply config.json]: This will mess up the git blame log
